### PR TITLE
Prepare base ConstantinopleVM structure

### PR DESCRIPTION
**Pull request by cburgdorf**
### What was wrong?

As stated in #1107 we don't have the base structure for the Constantinople fork yet.

### How was it fixed?

Adds all the base structure to support the Constantinople fork. Notice that this doesn't yet add the `ConstantinopleVM` to any chain though.

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

File: eth/vm/forks/constantinople/blocks.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from eth.vm.forks.byzantium.blocks import (
+    ByzantiumBlock,
+)
+
+from .transactions import (
+    ConstantinopleTransaction,
+)
+
+
+class ConnstantinopleBlock(ByzantiumBlock):
```
Comment (veox) [inline eth/vm/forks/constantinople/blocks.py]: Typo: Co**nn**stantinople (double "n").

File: eth/vm/forks/constantinople/transactions.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+from eth.utils.transactions import (
+    create_transaction_signature,
+)
+
+
+class ConstantinopleTransaction(ByzantiumTransaction):
+    @classmethod
+    def create_unsigned_transaction(cls, nonce, gas_price, gas, to, value, data):
+        return ByzantiumUnsignedTransaction(nonce, gas_price, gas, to, value, data)
```
Comment (carver) [inline eth/vm/forks/constantinople/transactions.py]: Should probably be `ConstantinopleUnsignedTransaction`

File: eth/vm/forks/__init__.py [hunk 3]
```diff
@@ -13,3 +13,6 @@
 from .byzantium import (  # noqa: F401
     ByzantiumVM,
 )
+from .constantinople import (  # noqa: F401
+    ConstantinopleVM
+)
```
Comment (carver) [inline eth/vm/forks/__init__.py]: *nitpick* add trailing newline

File: eth/vm/forks/constantinople/transactions.py (hunk 2 above)
Comment (cburgdorf) [reply eth/vm/forks/constantinople/transactions.py]: :eagle: :eye:

**Pull request merged**